include/vcpkg/packagespec.h

```diff
@@ -145,16 +145,40 @@ namespace vcpkg
 
     struct Dependency
     {
+        struct Feature
+        {
+            std::string name;
+            PlatformExpression::Expr platform;
+            Feature(std::string name) : Feature(std::move(name), PlatformExpression::Expr::Empty()) { }
+            Feature(std::string name, PlatformExpression::Expr platform)
+                : name(std::move(name)), platform(std::move(platform))
+            {
+                Checks::check_exit(VCPKG_LINE_INFO,
+                                   !this->name.empty() && this->name != "core" && this->name != "default");
```

Comment:
I think this is technically a breaking change: as far as I know the word 'default' is a valid name of a feature.

Reply:
Afaik not: https://github.com/microsoft/vcpkg-tool/blob/439f07a5474f0bfc9509e4d4ec338958e55de76c/src/vcpkg/base/json.cpp#L1108

Reply:
🤔🤔🤔🤔🤔